infrastructure/tooling/src/smoketest/checks/role.js

```diff
@@ -0,0 +1,21 @@
+const taskcluster = require('taskcluster-client');
+
+exports.tasks = [];
+exports.tasks.push({
+  title: 'Create role and expand smoketest',
+  requires: [],
+  provides: [
+    'role-create-expand',
```

Comment:
A name starting with `target` will allow users to run just this test with `yarn smoketest --target=..`.  So, let's make this `target-roles`.